examples/apyfloat.py

```diff
@@ -9,8 +9,11 @@
 also convenience methods to do this either from a float or from a bit-pattern.
 """
 
+from IPython.core.interactiveshell import InteractiveShell
 from apytypes import APyFloat
 
+InteractiveShell.ast_node_interactivity = "all"  # Output from all lines
+
```

Comment:
I do not think that this is a good thing as it is not related to the example. And using this for all examples (by modifying something elsewhere) is not really a good idea either.

Reply:
I wasn't too pleased with the solution either, I'll remove it.
It came from me wanting to get both outputs from the added examples. Should I use `display()` instead or remove the "Which is equivalent to" part? Or split the example into two cells.